src/ResourceManager/LocationBasedServices/Commands.LocationBasedServices/Microsoft.Azure.Commands.LocationBasedServices.format.ps1xml

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+  <ViewDefinitions>
+    <View>
+      <Name>Microsoft.Azure.Commands.LocationBasedServices.Models.PSLocationBasedServicesAccount</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Commands.LocationBasedServices.Models.PSLocationBasedServicesAccount</TypeName>
+      </ViewSelectedBy>
+      <TableControl>
+        <TableHeaders>
+          <TableColumnHeader/>
+          <TableColumnHeader/>
+          <TableColumnHeader/>
+        </TableHeaders>
+        <TableRowEntries>
+          <TableRowEntry>
+            <TableColumnItems>
+              <TableColumnItem>
+                <PropertyName>ResourceGroupName</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>AccountName</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>Id</PropertyName>
```

Comment:
No other data that the user would liek to know about the account?

Reply:
At this time, our accounts themselves are dead simple.   Nothing else interesting (at least that's exposed in the SDK... yet)